doc/user_guide.md

```diff
@@ -0,0 +1,324 @@
+# User guide
+
+The guide will go over the deployment of the Exasol Azure helper Functions briefly and then focus on using them in azure data factory itself.
+
+## Requirements
+
+An azure subscription with sufficient privileges to create resource groups, storage accounts, function apps and an azure data factory.
+
+## Deployment
+
+A full section on all deployment options available for Azure Functions can be found in the azure function docs under the how-to guides: [Zip push deployment for Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/deployment-zip-push)
+
+In this guide I'll elaborate on 1 of the easiest user friendly ways to set up azure functions. 
+
+The microsoft documentation on deploying azure functions is way more exhaustive so if this doesn't cover your needs you might find a better option there.
+
+Let's get started:
+
+### Zip Deployment using the Azure Portal and Azure CLI:
+
+Setting everything up in the azure portal:
```

Comment:
I would suggest keeping it unified and use title case for Azure services name across the documentation